pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -641,14 +644,15 @@ func (exp *FlowExporter) findFlowType(conn flowexporter.Connection) uint8 {
 }
 
 func (exp *FlowExporter) fillEgressInfo(conn *flowexporter.Connection) {
-	egressName, egressIP, _, err := exp.egressQuerier.GetEgress(conn.SourcePodNamespace, conn.SourcePodName)
+	egressName, egressIP, egressNodeName, err := exp.egressQuerier.GetEgress(conn.SourcePodNamespace, conn.SourcePodName)
 	if err != nil {
 		// Egress is not enabled or no Egress is applied to this Pod
 		return
 	}
 	conn.EgressName = egressName
 	conn.EgressIP = egressIP
-	klog.V(4).InfoS("Filling Egress Info for flow", "Egress", conn.EgressName, "EgressIP", conn.EgressIP, "SourcePodNamespace", conn.SourcePodNamespace, "SourcePodName", conn.SourcePodName)
+	conn.EgressNodeName = egressNodeName
+	klog.V(4).InfoS("Filling Egress Info for flow", "Egress", conn.EgressName, "EgressIP", conn.EgressIP, "EgressNode", conn.EgressNodeName, "SourcePodNamespace", conn.SourcePodNamespace, "SourcePodName", conn.SourcePodName)
```

Comment:
not introduced by this PR, but I have a small request to use `klog.KRef` for the Pod reference:
```
..., "sourcePod", klog.KRef(conn.SourcePodNamespace, conn.SourcePodName)
```